src/arch/Lights/LightsDriver_LinuxPacDrive.cpp

```diff
@@ -6,10 +6,8 @@
 #include "LightsDriver_LinuxPacDrive.h"
 
 #include <cstdint>
+#include <libusb/libusb.h>
```

Comment:
for libusb will be "not found" because proper way is "<libusb-1.0/libusb.h>" (/usr/include/libusb-1.0/libusb.h)

If libusb was found and include directory directly passed to compiler (via -I/usr/include/libusb-1.0) then we can use relative path and `#include <libusb.h>` and may be this is preferred way, idk.

Reply:
It seems like it was unclear to me whether it's preferable to use `<libusb-1.0/libusb.h>` or `<libusb/libusb.h>`, so I planned to leave it up to teejusb as I defer to him on stylistic concerns.